portugues/amigo/tecnologia.md

```diff
@@ -0,0 +1,3 @@
+[O seu amigo pede para parar de incomodar toda vez que tem um problema no celular.](incomodar/tecnologia.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.